0023-executor-primitive.md

```diff
@@ -0,0 +1,188 @@
+# Executor primitive
+
+| **Status**        | **Proposed/Accepted/Deprecated** |
+|:------------------|:---------------------------------------------|
+| **RFC #**         | 0023                                         |
+| **Authors**       | Salva de la Puente (salva@ibm.com),  Antonio Corcoles, Ian Hincks    |
+| **Deprecates**    | TBD                 |
+| **Submitted**     | 2025-07-30                                   |
+| **Updated**       | YYYY-MM-DD                                   |
+
+## Summary
+In Qiskit, the quantum computer interface is built around primitives, which are
+the fundamental operations carried by a quantum computer on a quantum circuit.
+Currently, Qiskit offers two such primitives: the Sampler and the Estimator.
+The Estimator leverages advanced error mitigation techniques to enhance the quality
+of results from noisy quantum computers. On a noiseless quantum computer, the
+Estimator could instead be constructed atop the Sampler, simplifying its role.
+However, given current noise levels, robust error mitigation is essential,
+requiring executing thousands or even millions of circuit variations to implement
+certain mitigation protocols. Transmitting these variations across cloud environments
```

Comment:
Here we should distinguish between the estimator abstraction, which says nothing about mitigation, and our IBM hardware implementation---the primitives service---that has options to enable it.